projects/kit/src/lib/processors/prefix-postprocessor.ts

```diff
@@ -4,19 +4,23 @@ export function maskitoPrefixPostprocessorGenerator(
     prefix: string,
 ): NonNullable<MaskitoOptions['postprocessor']> {
     return ({value, selection}, initialElementState) => {
-        const minCaretIndex = prefix.length;
-
-        if (value.length < minCaretIndex && initialElementState.value) {
-            return {selection: [minCaretIndex, minCaretIndex], value: prefix};
+        if (
```

Comment:
can we move it to semantic variable or fn?

```ts
return ({value, selection}, state) => {
  
     if (isAlreadyValidStartsWith(value, state)) {
         return {value, selection};
      }


```

Reply:
https://github.com/Tinkoff/maskito/pull/257#discussion_r1172342313